### PR TITLE
[hi] Added content/hi/docs/concepts/architecture/cloud-controller.md

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
Issue Number #39548

Localized en/docs/concepts/architecture/cloud-controller.md in hindi language :)

### CODE REVIEW

File: content/hi/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+रूट कंट्रोलर क्लाउड में रूट्स को कॉन्फ़िगर करने के लिए जिम्मेदार है
+उचित रूप से ताकि आपके कुबेरनेट्स में विभिन्न नोड्स पर कंटेनर हों
+क्लस्टर एक दूसरे के साथ संवाद कर सकते हैं।
+
+क्लाउड प्रदाता के आधार पर, रूट कंट्रोलर ब्लॉक भी आवंटित कर सकता है
+पॉड नेटवर्क के लिए IP पतों की संख्या।
+
+### सेवा नियंत्रक
+
+{{< glosary_tooltip text="Services" term_id="service" >}} क्लाउड के साथ एकीकृत
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/cloud-controller.md:71]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/cloud-controller.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+- Create
+- Patch
+- Update
+
+`v1/ServiceAccount`:
+
+- Create
+
+क्लाउड के लिए {{< glosary_tooltip term_id="rbac" text="RBAC" >}} ClusterRole
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/cloud-controller.md:136]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

**Comment by utkarsh-singh1** [comment]
Unavailable glossary sort code - cloud-controller-manager 

```
ERROR  [hi] "docs/concepts/architecture/cloud-controller.md": "cloud-controller-manager" is not a valid glossary term_id
```

**Comment by utkarsh-singh1** [comment]
Hi @akash-kumar-saw, Before you change term_id under glossary_definition, check [here](https://kubernetes.io/hi/docs/reference/glossary/?fundamental=true), if it's term_id is available in English use it in the original format.

File: content/hi/docs/concepts/architecture/cloud-controller.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+<!-- overview -->
+
+{{< feature-state state="beta" for_k8s_version="v1.11" >}}
+
+क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजी आपको सार्वजनिक, निजी और हाइब्रिड क्लाउड पर कुबेरनेट चलाने देती है।
+कुबेरनेट्स स्वचालित, एपीआई-संचालित बुनियादी ढांचे के बीच तंग युग्मन के बिना विश्वास करता है
+अवयव।
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/cloud-controller.md:15]: For `glossary_definition` to work as expected we should localize the glossary entry for "cloud-controller-manager" (i.e. [en/docs/reference/glossary/cloud-controller-manager.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/cloud-controller-manager.md)) and incorporate it into this pull request by adding to list of Hindi glossary items inside [/hi/docs/reference/glossary/cloud-controller-manager.md](https://github.com/kubernetes/website/tree/main/content/hi/docs/reference/glossary/).

Comment (dipesh-rawat) [reply content/hi/docs/concepts/architecture/cloud-controller.md:15]: @akash-kumar-saw As part of this pull request, you would also need to add the Hindi translation for the glossary entry `cloud-controller-manager` to complete the localization of this 'Cloud Controller Manager' page.

**Pull request closed without merging**